src/competitions/migrations/0013_enduser_email.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.2 on 2019-07-19 22:22
```

Comment:
Is this migration really needed, can't you just delete `0012_enduser.py` and run makemigrations again?

Reply:
delete: 0012_enduser.py,  0013_enduser_email.py, 0014_auto_20190720_0203.py, 0015_auto_20190720_0208.py  and then run makemigrations again

Reply:
Changed as requested